dd-sdk-android-core/BackgroundThreads.md

```diff
@@ -0,0 +1,103 @@
+# Background Threads
+
+This document lists all background threads created by the Datadog SDK (and its features and integrations),
+with their purposes, where they are created, and the stress testing done on each of them.
+
+### CoreFeature.persistenceExecutorService: `FlushableExecutorService`
+
+##### Purpose and Usage
```

Comment:
do you think it is useful to tell in this doc how many threads the particular service is managing?

Reply:
For now, I don't think it's relevant, this doc can evolve as our needs grow